src/main/java/uk/gov/homeoffice/digital/sas/balancecalculator/models/accrual/Contributions.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.homeoffice.digital.sas.balancecalculator.models.accrual;
+
+import java.math.BigDecimal;
+import java.util.Map;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Getter
+@Setter
```

Comment:
We have code smells regarding unused private fields. I think we can rectify this by limiting the getter and setter to specific fields.

Currently we don't access timeEntries & total

Reply:
See my response above re false positives